Sources/DeepLearning/Optimizer.swift

```diff
@@ -119,22 +127,26 @@ public class Adam<Model: Layer, Scalar: TensorFlowFloatingPoint>: Optimizer
 ///
 /// Reference: ["rmsprop: Divide the gradient by a running average of its recent magnitude"](
 /// http://www.cs.toronto.edu/~tijmen/csc321/slides/lecture_slides_lec6.pdf)
-public class RMSProp<Model: Layer, Scalar: TensorFlowFloatingPoint>: Optimizer
+public class RMSProp<Model: Layer>: Optimizer
     where Model.AllDifferentiableVariables == Model.CotangentVector {
     /// The learning rate.
-    public var learningRate: Scalar
+    public var learningRate: Float
     // TODO: Document `rho`. Keras doesn't document `rho`.
-    public var rho: Scalar
-    /// A small scalar added to the denominator to improve numerical stability.
-    public var epsilon: Scalar
+    public var rho: Float
+    /// A small Float added to the denominator to improve numerical stability.
```

Comment:
I don't think the comment needs to change.

Reply:
Reverted.